en/docs/integrate/develop/creating-artifacts/creating-scheduled-task.md

```diff
@@ -51,4 +51,4 @@ Update the task properties to specify the incoming message that should trigger t
 
 ## Tutorials
 
--   See the tutorial on [periodically executing an integration process]({{base_path}}/integrate/tutorials/using-scheduled-tasks) using a scheduled task
+-   See the tutorial on [periodically executing an integration process]({{base_path}}//tutorials/integration-tutorials/using-scheduled-tasks) using a scheduled task
```

Comment:
Please remove the additional slash before 'tutorials'

Reply:
fixed with [33b1850](https://github.com/wso2/docs-apim/pull/7321/commits/33b1850d15ef4f967024552b40b35169ff6bc33c)